processor/Cargo.toml

```diff
@@ -23,10 +23,12 @@ internals = ["miden-air/internals"]
 std = ["vm-core/std", "winter-prover/std"]
 
 [dependencies]
+thiserror = { version = "1.0", default-features = false }
 tracing = { version = "0.1", default-features = false, features = ["attributes"] }
-vm-core = { package = "miden-core", path = "../core", version = "0.8", default-features = false }
 miden-air = { package = "miden-air", path = "../air", version = "0.8", default-features = false }
+vm-core = { package = "miden-core", path = "../core", version = "0.8", default-features = false }
 winter-prover = { package = "winter-prover", version = "0.8", default-features = false }
```

Comment:
Seems like this is still using Winterfell v0.8 - should we not rebase it to work with Winterfell v0.9?